src/languages/en.js

```diff
@@ -269,6 +269,10 @@ export default {
         newPassword: 'New password',
         newPasswordPrompt: 'New password must be different than your old password, have at least 8 characters,\n1 capital letter, 1 lowercase letter, 1 number.',
```

Comment:
This error message seems wrong. If your problem is that the new password isn't different from the old password then that's what we should say. We're kind of mixing feedback here instead of telling the user exactly what's wrong.

Reply:
As a user, I'm going to wonder which is my problem? Passwords need to change? Or format is bad?

Reply:
Apologies, to clarify, mainly wanted to discuss this improvement and get some opinions before updating.